mm2src/coins/eth.rs

```diff
@@ -245,12 +246,51 @@ pub mod gas_limit {
     pub const ERC20_RECEIVER_SPEND: u64 = 150_000;
     /// Gas limit for swap refund tx with coins
     pub const ETH_SENDER_REFUND: u64 = 100_000;
-    /// Gas limit for swap refund tx with with ERC20 tokens
+    /// Gas limit for swap refund tx with ERC20 tokens
     pub const ERC20_SENDER_REFUND: u64 = 150_000;
     /// Gas limit for other operations
     pub const ETH_MAX_TRADE_GAS: u64 = 150_000;
 }
 
+/// Default gas limits for EthGasLimitV2
+pub mod gas_limit_v2 {
+    /// Gas limits for maker operations in EtomicSwapMakerV2 contract
+    pub mod maker {
+        pub const ETH_PAYMENT: u64 = 65_000;
+        pub const ERC20_PAYMENT: u64 = 115_000;
```

Comment:
This is lower than the legacy `ERC20_PAYMENT` limit, any reason for this and others? I will agree to keep these limits until tested but these should be the highest possible values and if a token requires lower, it's set in config.

Reply:
Hmmm re checked doc comment for legacy constant. here is a note about ERC-1967 proxied ERC20 contracts, which have higher gas usage than regular Erc20
```rust
    /// Gas limit for swap payment tx with ERC20 tokens
    /// real values are 98,9K for ERC20 and 135K for ERC-1967 proxied ERC20 contracts (use 'gas_limit' override in coins to tune)
    pub const ERC20_PAYMENT: u64 = 150_000;
```


the gas usage for v2 in Sepolia and Hardhat was like this:

| Func Name          | Signature   | Sepolia   | HardHat   |
|--------------------|-------------|-----------|-----------|
| erc20TakerPayment  | 0xD6A71EB4  | 75,008    | 92,108    |
|                    |             | 93,852    |           |

The highest gas usage for v2 erc20TakerPayment is a bit smaller than legacy (93,852 < 98,9K). Almost the same for maker  (92,220 < 98,9K).
But I didnt test ERC-1967 proxied ERC20. Probably will be the same situation. 
I suggest to follow the same approach for v2 erc20 payments and set default const as `150_000`. We can reduce it after tests, if necessary

Reply:
Increased erc20 payment default gas consts https://github.com/KomodoPlatform/komodo-defi-framework/pull/2211/commits/a4fd5d3f3ba5ce2abc30f374550605a4621e7109

Reply:
I think we still need to increase some other values.

Reply:
Adjusted gas limits https://github.com/KomodoPlatform/komodo-defi-framework/pull/2211/commits/6e42e34714b46a67bd241dbbccf7076f5a60c834